dev/secrets/custom-secret-key.yml

```
@@ -0,0 +1,7 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: 'custom-secret-key'
```

We shouldn't need quotes here (for consistency)
```suggestion
  name: custom-secret-key
```